test/jest/roktManager.spec.ts

```diff
@@ -215,5 +220,195 @@ describe('RoktManager', () => {
             expect(kit.selectPlacements).toHaveBeenCalledWith(options);
             expect(kit.launcher.selectPlacements).toHaveBeenCalledWith(options);
         });
+
+        it('should set sandbox to false in placement attributes when initialized as true', () => {
```

Comment:
should this be `true`?
```suggestion
        it('should set sandbox to true in placement attributes when initialized as true', () => {
```